### PR TITLE
Add unit tests to functionalities related to Shipping Labels M4: Add/create package

### LINKED ISSUE #4464: Add unit test related to Shipping Label project's add packages feature.
The following PRs adds functionalities related to activating/adding packages in Shipping Labels flow:

#4354 
#4397 
#4433
#4494 
#4471 

The added code there does not have related unit tests added yet, and they need to be added soon :)

### PR DESCRIPTION
**Pull request by hafizrahman**
To fix #4464

This PR adds unit tests to various functionalities added in the add/create package screen.

### To test:

Run unit tests, check especially for these files and make sure they all succeed:
- ShippingLabelCreatePackageViewModelTest.kt
- ShippingLabelCreateCustomPackageViewModelTest.kt
- ShippingLabelCreateServicePackageViewModelTest.kt

Also please check each added test's name and please let me know if anything's unclear and could be better named.

Update release notes:

- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreateCustomPackageViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    private fun populateFields() {
+        viewModel.onFieldTextChanged(packageToCreate.title, InputName.NAME)
+        viewModel.onFieldTextChanged(packageToCreate.dimensions.length.toString(), InputName.LENGTH)
+        viewModel.onFieldTextChanged(packageToCreate.dimensions.width.toString(), InputName.WIDTH)
+        viewModel.onFieldTextChanged(packageToCreate.dimensions.height.toString(), InputName.HEIGHT)
+        viewModel.onFieldTextChanged(packageToCreate.boxWeight.toString(), InputName.EMPTY_WEIGHT)
+    }
+
+    @Test
+    fun `When an invalid package name is entered, display error message`() {
```
Comment (kidinov) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreateCustomPackageViewModelTest.kt]: nitpick: naming: `when` - lower case. Also, should `then` after =) . Same in a couple of other tests

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreateCustomPackageViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+        viewModel.onFieldTextChanged("", InputName.EMPTY_WEIGHT)
+
+        assertThat(state!!.lengthErrorMessage).isEqualTo(R.string.shipping_label_create_custom_package_field_empty_hint)
+        assertThat(state!!.widthErrorMessage).isEqualTo(R.string.shipping_label_create_custom_package_field_empty_hint)
+        assertThat(state!!.heightErrorMessage).isEqualTo(R.string.shipping_label_create_custom_package_field_empty_hint)
+        assertThat(state!!.weightErrorMessage).isEqualTo(R.string.shipping_label_create_custom_package_field_empty_hint)
+    }
+
+    @Test
+    fun `when a zero package length, width, or height value is entered, then display error message`() {
```
Comment (kidinov) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreateCustomPackageViewModelTest.kt]: Please correct me if I'm wrong. Shouldn't it be `given a zero package length width, when value is entered, then display error message` and in other places similar?

**Review by kidinov — APPROVED** [review]
Looks good! Just maybe check the names of the tests to follow the convention

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreateCustomPackageViewModelTest.kt (hunk 2 above)
Comment (hafizrahman) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreateCustomPackageViewModelTest.kt]: Thanks for pointing that out. I like that separation with putting the values in `given` and the action in `when`.

**Comment by hafizrahman** [comment]
I'm merging this since this is already approved. Happy to make another PR with more changes if needed.

**Pull request merged**